compiler/ballerina-lang/src/main/java/org/ballerinalang/toml/parser/ManifestProcessor.java

```diff
@@ -170,15 +172,21 @@ public static Manifest parseTomlContentAsStream(InputStream inputStream) throws
      * @throws TomlException When the project block is invalid.
      */
     private static void validateManifestProject(Manifest manifest) throws TomlException {
-        if (null == manifest.getProject().getOrgName() || "".equals(manifest.getProject().getOrgName())) {
+        String orgName = manifest.getProject().getOrgName();
+        String version = manifest.getProject().getVersion();
+        if (null == orgName || "".equals(orgName)) {
             throw new TomlException("invalid Ballerina.toml file: cannot find 'org-name' under [project]");
         }
-        
-        if (null == manifest.getProject().getVersion() || "".equals(manifest.getProject().getVersion())) {
+
+        if (!COMPILE_BALLERINA_ORG && ("ballerina".equals(orgName))) {
+            throw new TomlException("invalid Ballerina.toml file: 'org-name' under [project] cannot be 'ballerina'");
```

Comment:
Can we say `invalid Ballerina.toml file: 'ballerina' is a reserved organization name.`

Reply:
Yes. Changed.